saleor/core/utils/random_data.py

```diff
@@ -988,6 +988,19 @@ def create_vouchers():
     else:
         yield "Value voucher already exists"
 
+    voucher, created = Voucher.objects.get_or_create(
+        code="WithoutName",
```

Comment:
This will be later displayed in the dashboard in demo / local environments:
![image](https://user-images.githubusercontent.com/5421321/75895941-9d635880-5e36-11ea-9ead-ca739767f231.png)

Do we want to show "WithoutName" there? Look like a fixture from our conftest. I would rather put some code that looks natural in the UI later e.g. something like "VCO9KV98LC" - I generated it using our voucher codes generator.